invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsField.js

```diff
@@ -71,7 +71,7 @@ class CreatibutorsFieldForm extends Component {
       error || (creatibutorsList === formikInitialValues && initialError);
 
     return (
-      <DndProvider backend={HTML5Backend}>
+      <DndProvider backend={HTML5Backend} context={window}>
```

Comment:
Hey, just for me to understand: did you encounter the issue? I am asking because I see [here](https://github.com/react-dnd/react-dnd/blob/7c88c37489a53b5ac98699c46a506a8e085f1c03/packages/react-dnd/src/core/DndProvider.tsx#L92C51-L92C57) that if no context passed then it should use by default window.

Reply:
Yes, we are re-arranging the deposit form, still using invenio-rdm-records' components directly. As soon as we add two components that use the `DndProvier` (Creatibutors and Licenses, for example) it raises the error.
We are building the form dynamically upon resource type selection, that might be the reason why we don't see the error on the "regular: deposit form.

I looked at the code before making the PR and I don't fully understand why it fails, you are right that if not set it should `window` already ... 🤷‍♂️

Reply:
You got it, https://gitlab.com/egabancho/invenio-ubiquity/-/tree/custom-deposit-form/site/up/assets/semantic-ui/js/up

Reply:
I checked briefly but I cannot detect something wrong without really digging into the internals of the library... I think it is worth trying to understand it before we assign `window` there and be afraid to remove it afterwards :)

Reply:
You're definitely right. I believe I jumped the gun on this one. Something is going on when I try to change the deposit form (and the value of `context` is indeed `window` already), but I'm not sure what.
I'm going to close this one, but I will report back what I find in case some poor soul has to go through the same process I am going through right now :wink:

Thanks @zzacharo!